qa/suites/rgw/verify/validater/valgrind.yaml

```diff
@@ -13,8 +13,9 @@ overrides:
       mon:
         mon osd crush smoke test: false
     valgrind:
-      mon: [--tool=memcheck, --leak-check=full, --show-reachable=yes]
       osd: [--tool=memcheck]
       mds: [--tool=memcheck]
+      # mon commented out due to: http://tracker.ceph.com/issues/38827
+      #mon: [--tool=memcheck, --leak-check=full, --show-reachable=yes]
```

Comment:
Might it be possible to keep a subset of these checks and just stop, for the time being, the one(s) that fails?